src/openbench/config.py

```diff
@@ -5709,6 +5709,32 @@ class EvalGroup:
         module_path="openbench.evals.cosafe_m2s",
         function_name="cosafe_m2s",
     ),
+    "tau_bench_retail": BenchmarkMetadata(
+        name="TauBench Retail",
+        description="Tau-bench dual-control customer support simulation in the retail domain.",
+        category="agents",
+        tags=["agents", "tools", "graded"],
+        module_path="openbench.evals.tau_bench",
+        function_name="tau_bench_retail",
+        is_alpha=False,
+    ),
+    "tau_bench_airline": BenchmarkMetadata(
+        name="TauBench Airline",
+        description="Tau-bench airline operations benchmark with simulated users and shared tools.",
+        category="agents",
+        tags=["agents", "tools", "graded"],
+        module_path="openbench.evals.tau_bench",
+        function_name="tau_bench_airline",
+        is_alpha=False,
+    ),
+    "tau_bench_telecom": BenchmarkMetadata(
+        name="TauBench Telecom",
+        description="Tau-bench telecom troubleshooting scenarios exercising complex tool flows.",
+        category="agents",
+        tags=["agents", "tools", "graded"],
+        module_path="openbench.evals.tau_bench",
+        function_name="tau_bench_telecom",
+        is_alpha=False,
```

Comment:
missing a closing paren here btw
```suggestion
        is_alpha=False,
    ),
```